### ISSUE TITLE
Help pane home ugly at narrow widths

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#12643]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : Any
    OS Version      : Any
    R Version       : Any

### Steps to reproduce the problem

Display the help pane (Ctrl+3) and resize the column to be narrow-ish.

### Describe the problem in detail

The content does not reflow at narrower widths and becomes increasingly ugly, and eventually starts to require horizontal scrolling. This is a violation of [WCAG 1.4.10 Reflow](https://www.w3.org/TR/WCAG21/#reflow).

![Screenshot 2023-01-27 at 15 50 07](https://user-images.githubusercontent.com/10569626/215221229-498946c9-86be-4535-8156-7df6e03429d5.png)

Caveat: all of the IDE's UI has extensive problems with this WCAG criteria, so fixing the help pane is a drop in the bucket. We will never be able to fully support this due to the nature of our UI. That said, in high-zoom scenarios (limited vision), having this pane continue working would still be a win.

### Describe the behavior you expected

This simple page should be responsive and handle narrower widths more appropriately.

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jonvanausdeln** [comment]
Verified with 2023.03.0-daily+280, Windows 11, Ubuntu 22

Helped pane flowed correctly for better visibility. Tested with standard rstudio help, and several different packages.
